math/genvector/inc/Math/GenVector/Plane3D.h

```diff
@@ -230,16 +250,71 @@ namespace Math {
       /**
          Normalize the normal (a,b,c) plane components
       */
-      void Normalize();
+      template< typename SCALAR = T >
+      typename std::enable_if< std::is_arithmetic<SCALAR>::value, void >::type
+      Normalize()
+      {
+        // normalize the plane
+        const SCALAR s = std::sqrt( fA*fA + fB*fB + fC*fC );
```

Comment:
Are we explicitly disabling Vc support here, by not omitting the `using namespace std;`?